exchange/bitswap/testnet/virtual.go

```diff
@@ -72,61 +72,7 @@ func (n *network) deliver(
 
 	n.delay.Wait()
 
-	nextPeer, nextMsg := r.ReceiveMessage(context.TODO(), from, message)
-
-	if (nextPeer == "" && nextMsg != nil) || (nextMsg == nil && nextPeer != "") {
-		return errors.New("Malformed client request")
-	}
-
-	if nextPeer == "" && nextMsg == nil { // no response to send
-		return nil
-	}
-
-	nextReceiver, ok := n.clients[nextPeer]
-	if !ok {
-		return errors.New("Cannot locate peer on network")
-	}
-	go n.deliver(nextReceiver, nextPeer, nextMsg)
```

Comment:
yeah, bitswap isnt a request/reply protocol, this was testing incorrect assumptions about it being that